site/docs/resources/migrating_to_0_10_0.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 3
+---
+
+# Migrating to 0.10.0 ⛷️
+
+v0.10.0 changed the syntax of `very_good create`.
+Previous usage received templates, these templates were now transformed into sub commands so in the
+future, we can have multiple templates for flutter apps, for example.
+
+Comparisons between the previous and the current syntax:
```

Comment:
Is something missing after the `:` here?

Reply:
I think it's just all of the content that follows